components/Form/Button.tsx

```diff
@@ -1,4 +1,12 @@
-export const Button = ({ loading, children, type, ...props }) => (
+import { FC } from 'react';
+
+type Props = {
+  loading?: boolean;
+  children: string;
```

Comment:
Just some info for you. `FC`, usually written as `React.FC` USED TO have implicit ` children?: ReactNode `. The implicit children got removed in react 18 types, because it was quite a questionable decision to put it there in the first place and a lot of people don't use it for that exact reason.